policy/scan/local_scanner.go

```diff
@@ -568,6 +569,34 @@ func (s *localAssetScanner) run() (*AssetReport, error) {
 	return ar, nil
 }
 
+func noPolicyErr(availablePolicies []string, filter []string) error {
+	var sb strings.Builder
+	sb.WriteString("bundle doesn't contain any policies\n")
+	sb.WriteString("\n")
+
+	if len(availablePolicies) > 0 {
+		sb.WriteString("The following policies are available:\n")
+		for i := range availablePolicies {
+			policyMrn := availablePolicies[i]
+			sb.WriteString("- " + policyMrn + "\n")
+		}
+		sb.WriteString("\n")
+	} else {
+		sb.WriteString("The policy bundle for the asset does not contain any policies\n\n")
+	}
+
+	if len(filter) > 0 {
+		sb.WriteString("User selected policies that are allowed to run:\n")
+		for i := range filter {
+			policyMrn := filter[i]
+			sb.WriteString("- " + policyMrn)
```

Comment:
i think you might also want a newline here?

Reply:
Indeed